okhttp/src/test/java/okhttp3/BouncyCastleTest.kt

```diff
@@ -39,38 +32,10 @@ class BouncyCastleTest {
 
   @Before
   fun setUp() {
+    OkHttpDebugLogging.enable("org.bouncycastle.jsse")
     platform.assumeBouncyCastle()
   }
 
-  @Test
-  fun testTlsv13Works() {
-    platform.expectFailureOnJdkVersion(8)
```

Comment:
Test wasn't running previously and so didn't flag issues with client and server running together.